pkg/daemon/update.go

```diff
@@ -161,7 +163,12 @@ func (dn *Daemon) drain() error {
 		}
 		return errors.Wrap(err, "failed to drain node")
 	}
+
 	dn.logSystem("drain complete")
+	t := time.Since(startTime).Seconds()
+	glog.Infof("Successful drain took %v sec", t)
```

Comment:
per mcd logs, this logging/block is being hit:
```
I1024 17:58:51.277698    3826 update.go:1044] drain complete
I1024 17:58:51.280250    3826 update.go:169] Successful drain took 32.941232105 sec
```
but the metric never gets updated. wondering if it's bc of timing? it would update <.2 sec before the node reboots?

Reply:
so what i think is happening is that the scrape is taking longer to happen than the actual reboot. so if the scrape is 1 sec and the metric update to reboot is .2 sec then it never picks up the metric.  I _will_ pick it up if something post drain delays the reboot, which could still be helpful to have when trouble shooting, ie "well we know drain did complete at X seconds so what happened after that?"